### DISCUSSION TITLE
generateNodesFromDOM is not detect the list of type "check"

### DISCUSSION THREAD

**Discussion by edwinpgm** [facebook/lexical#3854] [Q&A]
Hi all! I have this html string and I want to transform to nodes, but when I try to use $generateNodesFromDOM is not detect that the list is type "check", instead of it use "bullet" as type.

This is the html:

```
<ul class="PlaygroundEditorTheme__ul1"><li role="checkbox" tabindex="-1" aria-checked="false" value="1" class="PlaygroundEditorTheme__listItem PlaygroundEditorTheme__listItemUnchecked"><span>Test 1</span></li></ul>
```

This is the result generated

<img width="165" alt="image" src="https://user-images.githubusercontent.com/3245450/217597688-20c19b1b-00d2-44c2-ba9b-9ae16bec513b.png">

Please someone could help me with this issue.

**Comment by acywatson** [comment]
Easiest thing for you to do right now is pre-process the HTML such that the ul tag contains the data attribute '__lexicallisttype="check"'. This will cause the editor to interpret the ul as a checklist.

https://github.com/facebook/lexical/blob/fad90c6132c6c1fcbb55d31fa5bd3b54868086f8/packages/lexical-list/src/LexicalListNode.ts#L291
